src/sentry/static/sentry/app/components/stream/groupRowActions.tsx

```diff
@@ -127,8 +127,8 @@ class GroupRowActions extends React.Component<Props> {
                         ? ((project as Project).latestRelease as Release)
                         : undefined
                     }
-                    orgId={orgId}
-                    projectId={group.project.id}
+                    orgSlug={orgSlug}
+                    projectSlug={group.project.slug}
```

Comment:
this line was the cause of the bug, the other changes should all be just renamed props